plugins/BotTracking/RecordBuilders/AIAssistantReports.php

```diff
@@ -216,4 +230,92 @@ private function populateTableForActionType(array $tables, int $actionType, LogA
             ]);
         }
     }
+
+    public function populateRequestTableForActionType(DataTable $table, int $actionType, LogAggregator $logAggregator): void
+    {
+        $where    = $logAggregator->getWhereStatement('bot', 'server_time');
+        $bindBase = $logAggregator->getGeneralQueryBindParams();
+
+        $sql = sprintf(
+            "SELECT log_action.name AS url, log_action.url_prefix, COUNT(*) AS requests
+             FROM %s AS bot
+             INNER JOIN %s AS log_action ON log_action.idaction = bot.idaction_url
+             WHERE log_action.name IS NOT NULL
+               AND log_action.name <> ''
+               AND log_action.type = %d
+               AND bot.bot_type = ?
+               AND %s
+             GROUP BY log_action.name
+             ORDER BY requests DESC, log_action.name",
+            BotRequestsDao::getPrefixedTableName(),
+            Common::prefixTable('log_action'),
+            $actionType,
+            $where
+        );
+
+        $resultSet = Db::query($sql, array_merge([BotDetector::BOT_TYPE_AI_ASSISTANT], $bindBase));
+
+        while ($record = $resultSet->fetch()) {
+            /**
+             * @var array{requests: int, url: string, url_prefix: ?int} $record
+             */
+            $path = ArchivingHelper::getActionExplodedNames($record['url'], $actionType, $record['url_prefix']);
+            [$row, $level] = $table->walkPath($path, [Metrics::COLUMN_REQUESTS => 0], $this->maxRowsInSubtable);
+
+            if ($row) {
+                $row->setColumn(Metrics::COLUMN_REQUESTS, $record['requests']);
+            }
+        }
+    }
+
+    /**
+     * @param array<string, DataTable> $tables
+     */
+    private function populateNumerics(ArchiveProcessor $archiveProcessor, array &$tables): void
+    {
+        $logAggregator = $archiveProcessor->getLogAggregator();
+
+        $table      = BotRequestsDao::getPrefixedTableName();
+        $visitTable = Common::prefixTable('log_visit');
+
+        $where = $logAggregator->getWhereStatement('bot', 'server_time');
+
+        $sql = <<<SQL
+SELECT
+    COUNT(*) AS requests,
+    SUM(CASE WHEN bot.http_status_code = 404 THEN 1 ELSE 0 END) AS not_found_requests,
```

Comment:
Should we also include `410 GONE` as that is usually indicating "not found, and do not try again"?